### ISSUE TITLE
apoc.refactor.mergeNodes reverses relationship direction of deleted nodes

### ISSUE THREAD

**Issue by will-hickson** [neo4j-contrib/neo4j-apoc-procedures#590]
when i run the refactor.mergeNodes the direction of the relationships to the merged node is now reversed. So before:
```
(a1)-{:HAS_REL]->(b1)
(a2)-{:HAS_REL]->(b2)
(a3)-{:HAS_REL]->(b3)
(a4)-{:HAS_REL]->(b4)
```
Then I merge b2 thru b4 into b1 and now:
```
(a1)-{:HAS_REL]->(b1)
(a2)<-{:HAS_REL]-(b1)
(a3)<-{:HAS_REL]-(b1)
(a4)<-{:HAS_REL]-(b1)
```
First raised on slack channel
https://neo4j-users.slack.com/archives/C08J1EZCH/p1505161266000042
reply from Andrew Bowman
https://neo4j-users.slack.com/archives/C08J1EZCH/p1505161924000157

**Comment by mihairaulea** [comment]
Hi Will, take a look at this PR, please: https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/592 . It seems the issue does not reproduce.

**Comment by jexp** [comment]
@will-hickson could you please check the suggestion from @mihairaulea ?

**Comment by will-hickson** [comment]
Hi @mihairaulea  I ran your  code and got the expected error

![image](https://user-images.githubusercontent.com/29492207/30540055-75c7fb44-9c43-11e7-84c7-d259e58249cb.png)

Python py2neo on Community Edition 3.2.1

**Comment by conker84** [comment]
closed via eeed4e1
